inputFiles/wellbore/CasedThermoElasticWellbore_ImperfectInterfaces_base.xml

```diff
@@ -0,0 +1,583 @@
+
+    <!-- SPHINX_SolversSurfaceGenerator -->
+    <SurfaceGenerator
+      name="SurfaceGen"
+      fractureRegion="Fracture"
+      targetRegions="{ casing, cement, rock }"
+      rockToughness="1.0e6"
+      mpiCommOrder="1"/>
+    <!-- SPHINX_SolversSurfaceGeneratorEnd -->
+  </Solvers>
+
+  <NumericalMethods>
+    <FiniteElements>
+      <FiniteElementSpace
+        name="FE1"
+        order="1"/>
+    </FiniteElements>
+
+    <FiniteVolume>
+      <TwoPointFluxApproximation
+        name="tpfaFlow"/>
+    </FiniteVolume>
+  </NumericalMethods>
+
+  <ElementRegions>
+    <!-- SPHINX_SurfaceElementRegion -->   
+    <SurfaceElementRegion
+      name="Fracture"
+      faceBlock="faceElementSubRegion"
+      materialList="{ fluid, fractureFilling, fractureContact, contactThermalCond }"
+      defaultAperture="5.e-3"/>
+    <!-- SPHINX_SurfaceElementRegionEnd -->  
+
+    <CellElementRegion
+      name="casing"
+      cellBlocks="{ casing }"
+      materialList="{ fluid, casing, casingThermalCond }"/>
+
+    <CellElementRegion
+      name="cement"
+      cellBlocks="{ cement }"
+      materialList="{ fluid, cement, cementThermalCond }"/>
+
+    <CellElementRegion
+      name="rock"
+      cellBlocks="{ rock }"
+      materialList="{ fluid, rock, rockThermalCond }"/>
+  </ElementRegions>
+
+  <Constitutive>
+    <Coulomb
+      name="fractureContact"
+      cohesion="0"
+      frictionCoefficient="0.5"
+      penaltyStiffness="1.0e8"
+      apertureTableName="apertureTable"/>
+
+    <!-- SPHINX_ContactFlowProperties -->
+    <CompressibleSolidParallelPlatesPermeability
+      name="fractureFilling"
+      solidModelName="nullSolid"
+      porosityModelName="fracturePorosity"
+      permeabilityModelName="fracturePerm"
+      solidInternalEnergyModelName="rockInternalEnergy"/>
+
+    <NullModel
+      name="nullSolid"/>
+
+    <PressurePorosity
+      name="fracturePorosity"
+      defaultReferencePorosity="1.00"
+      referencePressure="0.0"
+      compressibility="0.0"/>
+
+    <ParallelPlatesPermeability
+      name="fracturePerm"/>
+    <!-- SPHINX_ContactFlowPropertiesEnd -->
+
+    <PorousElasticIsotropic
+      name="casing"
+      solidModelName="casingSolid"
+      porosityModelName="casingPorosity"
+      permeabilityModelName="casingPerm"
+      solidInternalEnergyModelName="casingInternalEnergy"/>
+
+    <PorousElasticIsotropic
+      name="cement"
+      solidModelName="cementSolid"
+      porosityModelName="cementPorosity"
+      permeabilityModelName="cementPerm"
+      solidInternalEnergyModelName="cementInternalEnergy"/>
+
+    <PorousElasticIsotropic
+      name="rock"
+      solidModelName="rockSolid"
+      porosityModelName="rockPorosity"
+      permeabilityModelName="rockPerm"
+      solidInternalEnergyModelName="rockInternalEnergy"/>
+
+    <!-- SPHINX_PoroElasticProperties -->
+    <BiotPorosity
+      name="casingPorosity"
+      defaultReferencePorosity="1e-6"
+      grainBulkModulus="159.4202899e9"/>
```

Comment:
```suggestion
      grainBulkModulus="1.6e11"/>
```

Reply:
This unrounded value of the bulk modulus is computed from rounded values of Young modulus and Poisson ratio: E=220e9 Pa and nu=0.27

Reply:
I propose to keep it as it is to obtain equivalent results of the analytical method.